types/common-types/index.d.ts

```diff
@@ -17,15 +17,10 @@ declare namespace AutoLatexCommon {
         /**
          * Given a size and a cursor right before an equation, call function to undo the image within delimeters. Returns success indicator.
          */
-        editEquations(app: IntegratedApp, sizeRaw: string, delimiter: string): (1 | -1 | -2 | -4 | -3);
+        editEquations(app: IntegratedApp, sizeRaw: string, delimiter: string): (-2 | 1 | -1 | -4 | -5 | -3);
```

Comment:
It might be useful to change this return system to use enums or flags instead of this esoteric numbering system for the future :)